pkg/controller/volume/expand/expand_controller.go

```diff
@@ -368,8 +368,8 @@ func (expc *expandController) getPersistentVolume(ctx context.Context, pvc *v1.P
 }
 
 // isNodeExpandComplete returns true if  pvc.Status.Capacity >= pv.Spec.Capacity
-func (expc *expandController) isNodeExpandComplete(pvc *v1.PersistentVolumeClaim, pv *v1.PersistentVolume) bool {
-	klog.V(4).Infof("pv %q capacity = %v, pvc %s capacity = %v", pv.Name, pv.Spec.Capacity[v1.ResourceStorage], pvc.ObjectMeta.Name, pvc.Status.Capacity[v1.ResourceStorage])
+func (expc *expandController) isNodeExpandComplete(logger klog.Logger, pvc *v1.PersistentVolumeClaim, pv *v1.PersistentVolume) bool {
+	logger.V(4).Info("pv and pvc capacity", "pvName", pv.Name, "pvCapacity", pv.Spec.Capacity[v1.ResourceStorage], "pvcName", pvc.ObjectMeta.Name, "pvcCapacity", pvc.Status.Capacity[v1.ResourceStorage])
```

Comment:
"pvName" and "pvcName" can be replaced with "PV" and "PVC", also in some other places.

Reply:
done